bin/reacli.js

```diff
@@ -20,6 +20,7 @@ const createElement = async ({ firstParam = null, pathsToComponentsToCreate = []
 		if (validatePath(path) && validateName(path)) {
 			try {
 				switch (firstParam) {
+				case "c":
```

Comment:
I don't really like this way of `switch - case` hack, but I believe there was no other way 😉

Do you mind fixing https://github.com/reacli/cli/issues/73 before I can merge this PR ? Thank you in advance !

Reply:
Hey man, Thanks for the reply. Of course i'll do the change, but I would like know why don't you like `swich - case`? Is there some special reason? I would really like know it

Reply:
Truly, not technological reason. I felt it would be more readable to use `if - else` rather than `switch - casecasecase` 🙂